app/models/metadata-value-search.ts

```diff
@@ -0,0 +1,26 @@
+import { AsyncHasMany, AsyncBelongsTo, attr, belongsTo, hasMany } from '@ember-data/model';
+
+import MetadataPropertySearchModel from './metadata-property-search';
+import { SearchFilter } from './metadata-record-search';
+import SearchResultModel from './search-result';
+import OsfModel from './osf-model';
+
+export default class MetadataValueSearchModel extends OsfModel {
```

Comment:
Hmmm. I wonder how true it is that these would be extending OsfModel. There's a lot of stuff on there that's not likely to be implemented in the new API, like Sparse Fieldsets and the things that take advantage of them.